index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Matee Safi - Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <section class="header">
+      <header class="nav">
+        <nav>
+            <div class="logo-container">
+                <label class="logo-text">Matee Safi</label>
+            </div>
+            <div class="menu">
+                <img id="hamburger" src="./assets/hamburger-icon.svg" alt="hamburger menu">
+            </div>
+        </nav>
+      </header>
+    </section>
+    <section class="headline">
+        <div class="main-line">
+            <h1 class="orange">Hey There.<br>I'm Matee</h1>
+        </div>
+        <div class="second-line">
+            <h3 class="orange">I am a Software Developer</h3>
+        </div>
+        <div class="third-line">
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+        </div>
+        <div class="social-media">
+            <img class="icon" src="./assets/github-icon.png" alt="github">
+            <img class="icon" src="./assets/linkedin-icon.png" alt="linkedin">
+            <img class="icon" src="./assets/angelist-icon.png" alt="angelist">
+            <img class="icon" src="./assets/twitter-icon.png" alt="twitter">
+            <img class="medium" src="./assets/medium-icon.png" alt="medium">
+        </div>
```

Comment:
- Your list of social icons will be better if a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).
Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href. 

Can we kindly refactor your social icons to look something like this 👇🏽 

```
        <ul class="social-media">
            <li><a href="https:yourgithub"><img class="icon" src="./assets/github-icon.png" alt="github"></a></li>
            <li><a href="https:yourgithub"><img class="icon" src="./assets/linkedin-icon.png" alt="linkedin"></a></li>
            <li><a href="https:yourgithub"><img class="icon" src="./assets/angelist-icon.png" alt="angelist"></a></li>
            <li><a href="https:yourgithub"><img class="icon" src="./assets/twitter-icon.png" alt="twitter"></a></li>
            <li><a href="https:yourgit"><img class="medium" src="./assets/medium-icon.png" alt="medium"></a></li>
        </ul>
```

What do you think about this? 🤔

Reply:
@Favourezeugwa 
![Screenshot 2023-01-13 04:36:48](https://user-images.githubusercontent.com/112982412/212302529-73165d41-d3a0-468c-86ff-8b7b24ef64d2.png)

I've fixed the anchor tag problem but i have a problem with the ul tag.